src/actions/acceptBid.ts

```diff
@@ -36,17 +40,22 @@ export const acceptBid = async (
 
   const zAuction = await getZAuctionContract(signer, zAuctionAddress);
 
+  // v2 bids use WILD
+  // v2.1 bids use bid.bidToken
+  const bidToken = bid.bidToken ?? config.wildTokenAddress;
```

Comment:
v2 should never get to this point. Should throw an error instead of trying to accept using `acceptBidV2`